host/xtest/aes_perf.c

```diff
@@ -148,33 +150,23 @@ static const char *mode_str(uint32_t mode)
 static void usage(const char *progname, int keysize, int mode,
 				size_t size, int warmup, unsigned int l, unsigned int n)
 {
-	fprintf(stderr, "AES performance testing tool for OP-TEE (%s)\n\n",
-		TO_STR(VERSION));
-	fprintf(stderr, "Usage:\n");
-	fprintf(stderr, "  %s -h\n", progname);
-	fprintf(stderr, "  %s [-v] [-m mode] [-k keysize] ", progname);
-	fprintf(stderr, "[-s bufsize] [-r] [-i] [-n loops] [-l iloops] \n");
-	fprintf(stderr, "[-w warmup_time]\n");
+	fprintf(stderr, "Usage: %s [OPTIONS]\n", progname);
+	fprintf(stderr, "AES performance testing tool for OP-TEE\n");
+	fprintf(stderr, "\n");
 	fprintf(stderr, "Options:\n");
-	fprintf(stderr, "  -h    Print this help and exit\n");
-	fprintf(stderr, "  -i    Use same buffer for input and output (in ");
-	fprintf(stderr, "place)\n");
-	fprintf(stderr, "  -k    Key size in bits: 128, 192 or 256 [%u]\n",
-			keysize);
-	fprintf(stderr, "  -l    Inner loop iterations (TA calls ");
-	fprintf(stderr, "TEE_CipherUpdate() <x> times) [%u]\n", l);
-	fprintf(stderr, "  -m    AES mode: ECB, CBC, CTR, XTS [%s]\n",
-			mode_str(mode));
-	fprintf(stderr, "  -n    Outer loop iterations [%u]\n", n);
-	fprintf(stderr, "  -r    Get input data from /dev/urandom ");
-	fprintf(stderr, "(otherwise use zero-filled buffer)\n");
-	fprintf(stderr, "  -s    Buffer size (process <x> bytes at a time) ");
-	fprintf(stderr, "[%zu]\n", size);
-	fprintf(stderr, "  -v    Be verbose (use twice for greater effect)\n");
-	fprintf(stderr, "  -w    Warm-up time in seconds: execute a busy ");
-	fprintf(stderr, "loop before the test\n");
-	fprintf(stderr, "        to mitigate the effects of cpufreq etc. ");
-	fprintf(stderr, "[%u]\n", warmup);
+	fprintf(stderr, "  -h|--help Print this help and exit\n");
+	fprintf(stderr, "  -v        Be verbose (use twice for greater effect)\n");
+	fprintf(stderr, "\n");
```

Comment:
i sorted them by "topics". at least I tried to.
I will revert to alphabetical order.